compiler/src/iree/compiler/Dialect/Modules/VMVX/Conversion/VMVXToVM/test/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ iree_add_all_subdirs()
 iree_lit_test_suite(
   NAME
     lit
+  SRCS
+    "microkernels.mlir"
```

Comment:
we'll want to split these up per op as we scale - debugging a single file with many patterns is hard